Tests/NIOIMAPCoreTests/Parser/Grammar/GrammarParser+Fetch+Tests.swift

```diff
@@ -69,6 +71,7 @@ extension GrammarParser_Fetch_Tests {
                 ("FLAGS (\\seen)", " ", .simpleAttribute(.flags([.seen])), #line),
                 ("FLAGS (\\seen \\answered \\draft)", " ", .simpleAttribute(.flags([.seen, .answered, .draft])), #line),
                 (")\r\n", " ", .finish, #line),
+                ("PREVIEW \"Lorem ipsum dolor sit amet\"", " ", .simpleAttribute(.preview(.init("Lorem ipsum dolor sit amet"))), #line),
```

Comment:
This should have a test case for `NIL` and also for the the literal case (if possible).

Here’s an example from the RFC:
```
  S: * 1 FETCH (RFC822.SIZE 5647 PREVIEW {200}
  S: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  S: Curabitur aliquam turpis et ante dictum, et pulvinar dui congue.
  S: Maecenas hendrerit, lorem non imperdiet pellentesque, nulla
  S: ligula nullam
  S: )
```